x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/shared/map_processor_type_to_form.tsx

```diff
@@ -246,210 +371,373 @@ export const mapProcessorTypeToDescriptor: MapProcessorTypeToDescriptor = {
         />
       );
     },
+    getDefaultDescription: ({ field }) =>
+      i18n.translate('xpack.ingestPipelines.processors.defaultDescription.grok', {
+        defaultMessage: 'Runs grok patterns on string value in "{field}"',
+        values: {
+          field,
+        },
+      }),
   },
   gsub: {
     FieldsComponent: Gsub,
     docLinkPath: '/gsub-processor.html',
     label: i18n.translate('xpack.ingestPipelines.processors.label.gsub', {
       defaultMessage: 'Gsub',
     }),
-    description: i18n.translate('xpack.ingestPipelines.processors.description.gsub', {
+    typeDescription: i18n.translate('xpack.ingestPipelines.processors.description.gsub', {
       defaultMessage: 'Uses a regular expression to replace field substrings.',
     }),
+    getDefaultDescription: ({ pattern, field, replacement }) =>
+      i18n.translate('xpack.ingestPipelines.processors.defaultDescription.gsub', {
+        defaultMessage: 'Replaces values matching "{pattern}" in "{field}" with "{replacement}"',
```

Comment:
Thanks! Will fix 👍🏻

Reply:
Actually, I just looked at the code it we accept an empty field because we can replace to '""' in this case so leaving the field blank is equivalent to providing the `""` value.